src/index.d.ts

```diff
@@ -5208,52 +5208,123 @@ declare namespace admin.machineLearning {
+     */
     filter?: string;
+
+    /** The number of results to return in each page. */
     pageSize?: number;
+
+    /** A token that specifies the result page to return. */
     pageToken?: string;
   }
 
   /** Response object for a listModels operation. */
   interface ListModelsResult {
+    /** A list of models in your project. */
     readonly models: Model[];
+
+    /** 
+     * A token you can use to retrieve the next page of results. If null, the
+     * current page is the final page.
+     */
     readonly pageToken?: string;
   }
 
   /**
-   * A TFLite Model output object
+   * A TensorFlow Lite Model output object
    */
   interface TFLiteModel {
+    /** The size of the model. */
     readonly sizeBytes: number;
 
+    /** The URI of the model in Cloud Storage. */
     readonly gcsTfliteUri?: string;
   }
 
   /**
    * A Firebase ML Model input object
    */
   interface ModelOptions {
+    /** A name for the model. This is the name you use from your app to load the model. */
     displayName?: string;
+
+    /** Tags for easier model management. */
     tags?: string[];
 
+    /** 
+     * An object containing the URI of the model in Cloud Storage.
+     *
+     * Example: `tfliteModel: { gcsTfliteUri: 'gs://your-bucket/your-model.tflite' }`
+     */
     tfliteModel?: {gcsTfliteUri: string;};
   }
 
   /**
    * A Firebase ML Model output object
    */
   interface Model {
+    /** The ID of the model. */
     readonly modelId: string;
+
+    /** The model's name. This is the name you use from your app to load the model. */
     readonly displayName: string;
+
+    /** The model's tags. */
     readonly tags?: string[];
+
+    /** The timestamp of the model's creation. */
     readonly createTime: string;
+
+    /** The timestamp of the model's most recent update. */
     readonly updateTime: string;
+
     readonly validationError?: string;
+
+    /** True if the model is published. */
     readonly published: boolean;
+
+    /**
+     * The ETag identifier of the current version of the model. This value
+     * changes whenever you update any of the model's properties.
+     */
     readonly etag: string;
+
+    /**
+     * The hash of the model's `tflite` file. This value changes only when
+     * you upload a new TensorFlow Lite model.
+     */
     readonly modelHash?: string;
+
+    /**
+     * True if the model is locked by a server-side operation. You can't make
+     * changes to a locked model. See {@link waitForUnlocked `waitForUnlocked()`}.
+     */
     readonly locked: boolean;
+
+    /** 
+     * Wait for the model to be unlocked.
+     *
+     * @param {number} maxTimeSeconds The maximum time in seconds to wait.
+     *
+     * @return {Promise<void>} A promise that resolves when the model is unlocked
+     *   or the maximum wait time has passed.
+     */
     waitForUnlocked(maxTimeSeconds?: number): Promise<void>;
 
+    /** The model's TensorFlow Lite model file. */
```

Comment:
Sort of. It's a TFLiteModel object which contains reference to the original TFLite file and metadata about it (sizeBytes)